### PR TITLE
Enable configurable syntax highlighting

### PR DESCRIPTION
**Pull request by skadyrov**
Make it possible define syntax highlight style using Android `style` declaration. `CodeEditorParameters` contains `codeEditorStyle` field that can be changed when used.

As an example, I added `AtomOneDark` style via style declaration:
screenshot_1542042888

Old Darcula:
screenshot_1542043070

### CODE REVIEW

File: library/src/main/java/com/github/ahmadaghazadeh/editor/processor/CodeEditTextView.kt [hunk 1]
```diff
@@ -134,11 +136,51 @@ class CodeEditTextView : AppCompatMultiAutoCompleteTextView {
...
+        val syntaxSymbols = ta.color(R.styleable.CodeEditTextView_syntaxSymbols, R.color.syntaxSymbols)
+        val syntaxBrackets = ta.color(R.styleable.CodeEditTextView_syntaxBrackets, R.color.syntaxBrackets)
+        val syntaxKeywords = ta.color(R.styleable.CodeEditTextView_syntaxKeywords, R.color.syntaxKeywords)
+        val syntaxMethods = ta.color(R.styleable.CodeEditTextView_syntaxMethods, R.color.syntaxMethods)
+        val syntaxStrings = ta.color(R.styleable.CodeEditTextView_syntaxStrings, R.color.syntaxStrings)
+        val syntaxComments = ta.color(R.styleable.CodeEditTextView_syntaxComments, R.color.syntaxComments)
+        ta.recycle()
+
+        return StyleParams(
+                selectedLinePaint = StylePaint(false, false, colorSelectedLine),
```
Comment (shockbytes) [inline library/src/main/java/com/github/ahmadaghazadeh/editor/processor/CodeEditTextView.kt]: I would change the constructor of the `StylePaint` class into 
```Kotlin
class StylePaint(color: Int? = null, antiAlias: Boolean = false, dither: Boolean = false)
```
I just checked and most of the booleans are always false anyway. So the constructor call of StylePaint would shrink down to just 
```Kotlin
selectedLinePaint = StylePaint(colorSelectedLine)
```
which looks more expressive to me than two boolean variables which are always set to false

**Comment by shockbytes** [comment]
Is the styling of the `ExtendedKeyboard` somehow related to the `CodeEditor` styling, or are these two separate stylings?

File: library/src/main/java/com/github/ahmadaghazadeh/editor/processor/CodeEditTextView.kt (hunk 1 above)
Comment (skadyrov) [reply library/src/main/java/com/github/ahmadaghazadeh/editor/processor/CodeEditTextView.kt]: good point 👍

**Comment by skadyrov** [comment]
@shockbytes currently they are styled separately. If you want a custom styling for the extended keyboard, you will need to pass in your own layout for the code editor, and you are in control of how to style it as that view will not belong to the library.

**Comment by skadyrov** [comment]
@shockbytes please take a look again

**Pull request merged**